modules/tinymce/src/plugins/quickbars/test/ts/browser/SelectionToolbarTest.ts

```diff
@@ -20,7 +21,7 @@ describe('browser.tinymce.plugins.quickbars.SelectionToolbarTest', () => {
     toolbar: false,
     menubar: false,
     base_url: '/project/tinymce/js/tinymce'
-  }, [ LinkPlugin, QuickbarsPlugin ], true);
+  }, [ LinkPlugin, QuickbarsPlugin, PageBreakPlugin ], true);
```

Comment:
Otherwise there'd be a failure on Chrome and Safari due to the pagebreak placeholder not being properly converted to the pagebreak img, which I assume is because the page break plugin wasn't being initialised. It makes sense to have the PageBreakPlugin be part of setup modules since we're using it in the hook. Not sure why this hasn't surfaced before, probably due to a change in the order / chunking of tests?